CMakeLists.txt

```diff
@@ -184,7 +167,7 @@ find_package(OpenSSL REQUIRED)
 function(qlever_target_link_libraries target)
     target_link_libraries(${target} ${ARGN} absl::flat_hash_map
             absl::flat_hash_set absl::strings absl::str_format ICU::uc
-            ICU::i18n OpenSSL::SSL OpenSSL::Crypto GTest::gtest GTest::gmock stxxl fsst)
+            ICU::i18n OpenSSL::SSL OpenSSL::Crypto GTest::gtest GTest::gmock stxxl fsst nlohmann_json::nlohmann_json)
```

Comment:
Was there an `include_directories` for `nlohmann_json` with the old variant of the code? 
This should now be unused and can be dropped if existing, please check.

Reply:
Yes, there was (within the populate clause) and it was removed.